views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -217,8 +257,9 @@ define([
                 self.updateControls();
             });
 
-            this.player.on('idle', function() {
```

Comment:
I didn't find other references to an 'idle' event so I presume this change is safe. But if it could still exist, you should probably keep the listener.

Reply:
It has been removed on purpose. Idle event is still triggered by the player object and this listener should not be played anymore in that case. Event cycle of audio element is weird at best